slice/Ice/Locator.ice

```diff
@@ -5,119 +5,67 @@
-        /// @param id The adapter id.
-        /// @param proxy The adapter proxy (a dummy direct proxy created by the adapter). The direct proxy contains the
-        /// adapter endpoints. The proxy can be null, typically during adapter deactivation.
-        /// @throws AdapterNotFoundException Raised if the adapter cannot be found, or if the locator only allows
-        /// registered adapters to set their active proxy and the adapter is not registered with the locator.
-        /// @throws AdapterAlreadyActiveException Raised if an adapter with the same id is already active.
-        ["amd"] idempotent void setAdapterDirectProxy(string id, Object* proxy)
-            throws AdapterNotFoundException, AdapterAlreadyActiveException;
-
-        /// Set the adapter endpoints with the locator registry.
-        /// @param adapterId The adapter id.
-        /// @param replicaGroupId The replica group id.
-        /// @param proxy The adapter proxy (a dummy direct proxy created by the adapter). The direct proxy contains the
-        /// adapter endpoints. The proxy can be null, typically during adapter deactivation.
-        /// @throws AdapterNotFoundException Raised if the adapter cannot be found, or if the locator only allows
-        /// registered adapters to set their active proxy and the adapter is not registered with the locator.
-        /// @throws AdapterAlreadyActiveException Raised if an adapter with the same id is already active.
-        /// @throws InvalidReplicaGroupIdException Raised if the given replica group doesn't match the one registered
-        /// with the locator registry for this object adapter.
-        ["amd"] idempotent void setReplicatedAdapterDirectProxy(string adapterId, string replicaGroupId, Object* proxy)
-            throws AdapterNotFoundException, AdapterAlreadyActiveException, InvalidReplicaGroupIdException;
-
-        /// Set the process proxy for a server.
-        /// @param id The server id.
-        /// @param proxy The process proxy. The proxy is never null.
-        /// @throws ServerNotFoundException Raised if the server cannot be found.
-        ["amd"] idempotent void setServerProcessProxy(string id, Process* proxy)
-            throws ServerNotFoundException;
-    }
-
-    /// This interface should be implemented by services implementing the <code>Ice::Locator interface</code>. It should
-    /// be advertised through an Ice object with the identity <code>'Ice/LocatorFinder'</code>. This allows clients to
-    /// retrieve the locator proxy with just the endpoint information of the service.
+    /// Provides access to a {@link Locator} object via a fixed identity.
+    /// A LocatorFinder is always registered with identity @c Ice/LocatorFinder. This allows clients to obtain the
+    /// associated Locator proxy with just the endpoint information of the object. For example, you can use the
+    /// LocatorFinder proxy `Ice/LocatorFinder:tcp -h somehost -p 4061` to get the Locator proxy
```

Comment:
`<tt>...</tt>` breaks Javadoc, so I switched to markdown and enabled markdown parsing in the Slice doxyfile.

Reply:
We should also enable it in the C++ Doxyfile.

Reply:
Was already enabled in the C++ Doxyfile.